testdata/picard/illumina/25T8B25T/sams_with_4M/indicies/CAATAGTC.sam

```diff
@@ -1,4 +1,4 @@
-@HD	VN:1N5	SO:queryname
+@HD	VN:1.6	SO:queryname
```

Comment:
Looks like an error in your search and replace?  (just spot-checked this one - maybe many more?)